libbeat/cmd/instance/beat_test.go

```diff
@@ -1,6 +1,6 @@
 // +build !integration
 
-package beat
+package instance
```

Comment:
If we have an `instance` package, it would kind of make sense to have this under the `beat` package?